foundations/html_css/block-and-inline.md

```diff
@@ -69,4 +72,4 @@ This section contains questions for you to check your understanding of this less
 
 This section contains helpful links to related content. It isn’t required, so consider it supplemental.
 
-*   [This tutorial](https://learnlayout.com/no-layout.html) is a little dated at this point, but its examples are clear. The first 6 slides cover the material we've seen so far.
+*   [This tutorial](https://learnlayout.com/no-layout.html) is a little dated at this point, but its examples are clear. Read the first 6 slides cover the material we've seen so far, Understand the use of `display: none` and which issue `max-width` property solve?
```

Comment:
I do not think the sentence you change reads very well. Also,  do not call out the use of `display: none` on this resource because the use of it is frowned upon for accessibility.